source/release-notes.txt

```diff
@@ -10,6 +10,29 @@ Release Notes
    :depth: 1
    :class: twocols
 
+|compass| 1.45.4
+----------------
+
+New Features:
+
+-  Connect in new window with connect split-button (:issue:`COMPASS-8473`)
+
+Bug Fixes:
+
+- Update maps from map tile API v2 to raster tile API v3 (:issue:`COMPASS-8981`)
+
+- Prevent losing changes in validation editor (:issue:`COMPASS-9003`)
+
+- Document preview overflow styles (:issue:`COMPASS-8997`)
+
+- Hang when the shell output reaches maxOutputLines
+  (:issue:`COMPASS-7567` and :issue:`MONGOSH-2021`)
+
+- Update driver and patch ``package-lock.json`` hoisting for ``mongosh`` packages
+
+`Full Changelog available on Github
+<https://github.com/mongodb-js/compass/compare/v1.45.3...v1.45.4>`__
```

Comment:
Just wanted to call out that this is currently showing as empty for me. It looks like the 1.45.4 branch hasn't been cut yet on GitHub, so I'm guessing once that happens this will work as expected.

Reply:
Confirming that this link now works, given that the branch has been cut.